include/language/en_us.lang.php

```diff
@@ -1559,6 +1559,7 @@
     'LBL_ADD_TO_PROSPECT_LIST_BUTTON_LABEL' => 'Add To Target List',
     'LBL_ADD_TO_PROSPECT_LIST_BUTTON_TITLE' => 'Add To Target List',
     'LBL_ADD_TO_PROSPECT_LIST_BUTTON_LABEL_ACCOUNTS_CONTACTS' => 'Add Contacts To Target List',
+    'LBL_SEND_CONFIRM_OPT_IN_EMAIL' => 'Send Confirm OptIn Email',
```

Comment:
Please avoid new words and look for consistency: the word is used in SuiteCRM now in some different forms: Opt In, Opt in and OptIn.
Don't know if also used as Opt-in

Reply:
Ok, which one do you prefer? :)

Reply:
I can't see that change. Are you referring to an other file?

Reply:
I will using "Opt In" anywhere because It is the most commonly used already.

Reply:
I'm not an english user... so feel free to pick one!
Usually I fund it funny that english language its so easy to create new concepts and words!
see: Confirmed opt-in (COI) / Double opt-in (DOI)
https://en.wikipedia.org/wiki/Opt-in_email

Reply:
I'm not an english user also, so every time I'm a little bit confused about the language texts :D